tagstudio/src/qt/widgets/preview_panel.py

```diff
@@ -499,31 +497,32 @@ def update_widgets(self):
                     )
 
                     # TODO: Do this somewhere else, this is just here temporarily.
-                    extension = os.path.splitext(filepath)[1][1:].lower()
                     try:
                         image = None
-                        if extension in IMAGE_TYPES:
-                            image = Image.open(filepath)
-                        elif extension in RAW_IMAGE_TYPES:
+                        if filepath.suffix in IMAGE_TYPES:
+                            image = Image.open(str(filepath))
+                        elif filepath.suffix in RAW_IMAGE_TYPES:
                             with rawpy.imread(filepath) as raw:
                                 rgb = raw.postprocess()
                                 image = Image.new(
                                     "L", (rgb.shape[1], rgb.shape[0]), color="black"
                                 )
-                        elif extension in VIDEO_TYPES:
+                        elif filepath.suffix in VIDEO_TYPES:
                             video = cv2.VideoCapture(filepath)
                             video.set(cv2.CAP_PROP_POS_FRAMES, 0)
                             success, frame = video.read()
                             frame = cv2.cvtColor(frame, cv2.COLOR_BGR2RGB)
                             image = Image.fromarray(frame)
 
                         # Stats for specific file types are displayed here.
-                        if extension in (IMAGE_TYPES + VIDEO_TYPES + RAW_IMAGE_TYPES):
+                        if filepath.suffix in (
+                            IMAGE_TYPES + VIDEO_TYPES + RAW_IMAGE_TYPES
+                        ):
                             self.dimensions_label.setText(
-                                f"{extension.upper()}  •  {format_size(os.stat(filepath).st_size)}\n{image.width} x {image.height} px"
+                                f"{filepath.suffix.upper()}  •  {format_size(os.stat(filepath).st_size)}\n{image.width} x {image.height} px"
```

Comment:
```suggestion
                                f"{filepath.suffix.upper()[1:]}  •  {format_size(os.stat(filepath).st_size)}\n{image.width} x {image.height} px"
```
Removes leading "." from filetype display